flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/result/MaterializedCollectBatchResult.java

```diff
@@ -53,11 +54,13 @@
 
 	private volatile boolean snapshotted = false;
 
-	public MaterializedCollectBatchResult(TypeInformation<Row> outputType, ExecutionConfig config) {
+	public MaterializedCollectBatchResult(RowTypeInfo outputType, ExecutionConfig config) {
 		this.outputType = outputType;
 
 		accumulatorName = new AbstractID().toString();
-		tableSink = new CollectBatchTableSink(accumulatorName, outputType.createSerializer(config));
+		tableSink =
```

Comment:
How about we change the return type of `configure` in `CollectBatchTableSink` to `CollectBatchTableSink`?